suite-native/atoms/src/Input/Input.tsx

```diff
@@ -73,12 +73,13 @@ const inputStyle = prepareNativeStyle(utils => ({
     ...utils.typography.body,
     alignItems: 'center',
     justifyContent: 'center',
-    height: INPUT_TEXT_HEIGHT + INPUT_WRAPPER_PADDING_VERTICAL,
+    height: INPUT_TEXT_HEIGHT,
     color: utils.colors.gray700,
-    lineHeight: 0,
     borderWidth: 0,
     flex: 1,
-    paddingTop: INPUT_WRAPPER_PADDING_VERTICAL,
```

Comment:
I am not sure about this, I introduced this padding because I wanted input to fill all possible space, so if you click anywhere it will focus. Please add some temp border to it and check if it's filling whole space from top to bottom

Reply:
I would solve this with hitSlop so I increased top slop 34a49f8a5a9bdc0551381d2eb7664cc5caeb470d. Your solution worked on iOS but did not show any text on Android.